Rubberduck.Inspections/Concrete/EmptyConditionBlockInspection.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Rubberduck.Inspections.Abstract;
+using Rubberduck.Parsing.Inspections.Abstract;
+using Rubberduck.Parsing.Inspections.Resources;
+using Antlr4.Runtime;
+using Antlr4.Runtime.Misc;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.Parsing;
+using Rubberduck.Parsing.VBA;
+using Rubberduck.Inspections.Results;
+
+namespace Rubberduck.Inspections.Concrete
+{
+    [Flags]
+    public enum ConditionBlockToInspect
+    {
+        NA = 0x0,
+        If = 0x1,
+        ElseIf = 0x2,
+        Else = 0x4,
+        All = If | ElseIf | Else
+    }
+
+    internal class EmptyConditionBlockInspection : ParseTreeInspectionBase
+    {
+        public EmptyConditionBlockInspection(RubberduckParserState state,
+                                            ConditionBlockToInspect BlockToInspect)
```

Comment:
Ninject has no idea how to create the inspection instance, `ConditionBlockToInspect` doesn't belong as a constructor parameter. The inspection needs to inspect them all anyway.